### ISSUE TITLE
SQLite database is not able to be written to by default Docker setup

### ISSUE THREAD

**Issue by EmberHeartshine** [mdshack/shotshare#18]
I got a 500 internal server error when attempting to register an account. I checked the logs in `/app/storage/logs` and found an exception:
```
[previous exception] [object] (PDOException(code: HY000): SQLSTATE[HY000]: General error: 8 attempt to write a readonly database at /app/vendor/laravel/framework/src/Illuminate/Database/Connection.php:580)
```
So I navigated to `/app/database/` to check and lo and behold, `database.sqlite` is owned by root:root. I chown'd to www-data:www-data and the user registration started working again.

I have no idea what the security implications of this are, but being unable to write to the database seems like an oversight.

**Comment by Raytle** [comment]
> I got a 500 internal server error when attempting to register an account. I checked the logs in `/app/storage/logs` and found an exception:
> 
> ```
> [previous exception] [object] (PDOException(code: HY000): SQLSTATE[HY000]: General error: 8 attempt to write a readonly database at /app/vendor/laravel/framework/src/Illuminate/Database/Connection.php:580)
> ```
> 
> So I navigated to `/app/database/` to check and lo and behold, `database.sqlite` is owned by root:root. I chown'd to www-data:www-data and the user registration started working again.
> 
> I have no idea what the security implications of this are, but being unable to write to the database seems like an oversight.

By the way, Gparted does the same thing when formatting drives. I wasted an hour or two before I finally realized what was going on. From some discussions I read, at least some of the Gparted devs chose the newly formatted drives must be "owned by root:root" hill to die on. Sooo.. now I use gnome-disk-utility. I'm not a dev, I prefer to avoid command line tools for tasks I seldom need to perform.

But refusing to give users an option to choose ownership when formatting a new drive seems inane to me.

**Comment by mdshack** [comment]
Thanks for raising this issue @EmberHeartshine. It looks like I broke this a few fixes ago when I was fixing something else :astonished:. A fix is included in [v1.5.1](https://github.com/mdshack/shotshare/releases/tag/v1.5.1), along with a fix to another issue I found with the example docker setup.
